Orange/widgets/unsupervised/owdistancemap.py

```diff
@@ -333,8 +341,8 @@ def __init__(self):
         self.annot_combo = gui.comboBox(
             self.controlArea, self, "annotation_idx", box="Annotations",
             callback=self._invalidate_annotations, contentsLength=12)
-        self.annot_combo.setModel(itemmodels.VariableListModel())
-        self.annot_combo.model()[:] = ["None", "Enumeration"]
+        self.order = (self.Names[self.Enumeration], ) + DomainModel.SEPARATED
```

Comment:
Why is this particular order stored and others are not?

Maybe it would be better if you constructed all models in `__init__` and the just use the correct one. Even better, those that are not domain models but have just fixed strings can be class attributes.

Reply:
Because `self.order` is used later in the code.

Reply:
So could be other orders. This one is not special.